target_chains/sui/scripts/pyth/pyth_deploy.ts

```diff
@@ -1,99 +1,204 @@
-/// Deploy Pyth to Sui testnet (devnet deploy can be done via CLI)
 import {
+  Connection,
+  Ed25519Keypair,
   fromB64,
   getPublishedObjectChanges,
+  JsonRpcProvider,
+  MIST_PER_SUI,
   normalizeSuiObjectId,
   RawSigner,
   TransactionBlock,
-  JsonRpcProvider,
-  Ed25519Keypair,
-  Connection,
 } from "@mysten/sui.js";
 import { execSync } from "child_process";
 
-import dotenv from "dotenv";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import {
+  DefaultStore,
+  getDefaultDeploymentConfig,
+  SuiChain,
+} from "contract_manager";
```

Comment:
did you mean to do something like this?

`} from "../../../../contract_manager/src/index";`

the code isn't compiling for me as is

Reply:
hmm, it seems to work after I run

`npx lerna run build in the root directory
`
do you happen to know why that is?